### PR TITLE
docs(playout): add a tutorial to enable AAC in liquidsoap

### PR DESCRIPTION
**Pull request by caveman99**
Documentation update regarding #2184

### CODE REVIEW

File: docs/admin-manual/stream-configuration.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ Setting a higher bitrate for your output stream will only benefit your listeners
 
 :::
 
+:::caution
+
+The version of the liquidsoap playout handler distributed in supported operating systems doesn't support AAC streaming output. If you want to stream AAC, you will need to replace the liquidsoap package with a version that supports AAC. See this [tutorial](./tutorials/setup-liquidsoap-aac-streaming.md) for more information.
```
Comment (paddatrapper) [inline docs/admin-manual/stream-configuration.md:22]: Be specific here - the issue is Liquidsoap version 1.4.3 shipped in Bullseye and 1.4.1 shipped in Focal. Other distros versions are not affected

File: docs/admin-manual/tutorials/setup-liquidsoap-aac-streaming.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+title: How to update liquidsoap to support AAC streaming
+---
+
+This tutorials walks you though the steps required to replace the liquidsoap package with a version that supports AAC streaming.
```
Comment (paddatrapper) [inline docs/admin-manual/tutorials/setup-liquidsoap-aac-streaming.md:5]: Add a warning at the top of this page talking about the lack of security support and the dangers of following this to get AAC working.

**Comment by paddatrapper** [comment]
This doesn't help Docker installs, but that would require something integrated into the image anyway

File: docs/admin-manual/tutorials/setup-liquidsoap-aac-streaming.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+:::warning
+
+Replacing the liquidsoap package has security implications, since this will remove the package from the system's package manager. This means that the package manager will not be able to update the liquidsoap package in the future. This includes backports of security fixes.
+
+Libretime is NOT compatible with Liquidsoap 2.x at the time of this writing. Future versions of Libretime will support Liquidsoap 2.x which will render these instructions obsolete.
```
Comment (jooola) [inline docs/admin-manual/tutorials/setup-liquidsoap-aac-streaming.md:11]: Closing fences might be missing here.

Comment (caveman99) [reply docs/admin-manual/tutorials/setup-liquidsoap-aac-streaming.md:11]: 'tis indeed. see new PR...

**Pull request merged**